app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -269,6 +271,22 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
                 }))
     }
 
+    fun getCustomFormsForAttendees(eventId: Long) {
+        val filter = "[{\"name\":\"form\",\"op\":\"like\",\"val\":\"order\"}]"
```

Comment:
why like? are you comparing values?

Reply:
The filter requires a `op` value actually. So I thought of using `like`.

Reply:
That is why the equality operator should be used